README.md

```diff
@@ -138,8 +138,15 @@ Whether to run unused export detection or not.
 #### options.log (default: `"all"`)
 
 `"all"`: show all messages.
+
 `"unused"`: only show messages when there are either unused files or unused export.
 
+`"none"`: won't show unused files or unused export messages in the terminal, it can keep terminal clean when set `exportJSON` to `true`
+
+#### options.exportJSON (default: `false`)
```

Comment:
should we support either boolean and path?

Reply:
Right, I will support this in the next two days.

Reply:
now it support boolean and path, Pls help to check with this.